src/ParcelRegistry.Projections.Extract/Migrations/20240517142535_AddCountOnParcelLink.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace ParcelRegistry.Projections.Extract.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddCountOnParcelLink : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropPrimaryKey(
```

Comment:
@ArneD I don't know why this change is added to the migration? More like these below.